lib/pages/signup/start-page.js

```diff
@@ -3,15 +3,20 @@
 import { By } from 'selenium-webdriver';
 
 import * as dataHelper from '../../data-helper';
+import * as driverHelper from '../../driver-helper';
 import AsyncBaseContainer from '../../async-base-container';
 
 export default class StartPage extends AsyncBaseContainer {
 	constructor( driver, url ) {
-		super( driver, By.css( '.step-wrapper' ), url );
+		const loggedOutThemesURL = dataHelper.getCalypsoURL( 'themes' );
```

Comment:
this is a SSR page that works logged out - I guess we could also use the log-in page